.changeset/swift-crabs-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vocab/phrase': patch
+---
+
+Pull translations from Phrase using the locale name rather than locale code
```

Comment:
I wonder if this should be described in terms of a "fix".

```suggestion
Fix issue where translations were pulled by locale code rather than locale name.

Translations have always been pushed by Phrase's locale name.
In Phrase, it's possible to set the code different to the name.
If this was done, Vocab would look for translations that matched the code and not the original name.
This typically resulted in no translations being found.

Vocab should now always push and pull based on Phrase's locale name regardless of any customised locale codes in Phrase.
```